src/scss/includes/modals/index.scss

```diff
@@ -60,13 +65,20 @@
 }
 
 .modal__button {
-  width: 271px;
+  width: 100%;
   height: 35px;
```

Comment:
Now that it's a standard button, you don't need any of these rules, except for `width`.